src/implementations/twig/components/table/table.html.twig

```diff
@@ -115,6 +120,9 @@
             {% elseif cell.label is not empty and _sortable %}
               {% set _heading_attribute = _heading_attribute ~ ' data-ecl-table-sort-toggle' %}
             {% endif %}
+            {% if cell.headers is not empty %}
+              {% set _heading_attribute = _heading_attribute ~ 'headers="' ~ cell.headers|e('html_attr') ~ '"' %}
```

Comment:
it seems it's correctly rendered, but for "readability" of the template i would add a white space between attributes.